lib/modules/dosomething/dosomething_reportback/dosomething_reportback.forms.inc

```diff
@@ -399,7 +399,7 @@ function dosomething_reportback_form_save_file(&$form_state) {
 function dosomething_reportback_image_edit($image, $x, $y, $width, $height, $rotate = NULL) {
   // Rotate the image and then crop it if rotation was successful.
   if ($rotate) {
-    image_rotate($image, $rotate);
+    $rotate_worked = image_rotate($image, $rotate, 0xffffff);
```

Comment:
Not needed for merge, but If this returns FALSE, it might be nice to use `drupal_set_message` to provide some info to the user.